src/components/ThreadsFeed/index.svelte

```diff
@@ -0,0 +1,74 @@
+<script lang="ts">
+    import { onMount } from "svelte";
+    import { getFeed } from "./request";
+    import ThreadItem from "../ThreadItem/item.svelte";
+
+    import type { Item } from "../ThreadItem/types";
+
+    export let endpoint: string;
+    export let communityId: string;
+
+    enum SortBy {
+        CREATED_AT = "CREATED_AT",
+        NET_UP_VOTES = "NET_UP_VOTES",
+    }
+
+    enum SortOrder {
+        ASC = "ASC",
+        DESC = "DESC",
+    }
+
+    export let limit: number = 5;
+    export let sortBy: SortBy = SortBy.NET_UP_VOTES;
```

Comment:
@markholdex @georgeciubotaru we need to make sure that the threads are sorted by date, not by upvotes amount. Because then we are going to end up with a static page where all top threads are going to take the higher passions and the content is going to be not dynamic.